src/expandableCalendar/index.tsx

```diff
@@ -427,8 +429,10 @@ class ExpandableCalendar extends Component<Props, State> {
   };
 
   onVisibleMonthsChange = (value: DateData[]) => {
-    if (this.visibleMonth !== first(value)?.month) {
-      this.visibleMonth = first(value)?.month; // equivalent to this.getMonth(value[0].dateString)
+    const month = first(value)?.month;
+    if (month && this.visibleMonth !== month) {
+      this.visibleMonth = month; // equivalent to this.getMonth(value[0].dateString)
+      this.visibleYear = first(value)?.year || this.getYear(new Date());
```

Comment:
this.visibleYear is already set on line 161. So why do we need to default to this.getYear(new Date())? Can't we just not set the value if it's undefined?

Reply:
Done :)